### DISCUSSION TITLE
Property 'propname' does not exist on type 'unknown'

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#15883] [Questions] [answered]
The problem was detected when I added vue-tsc (type-checking) on a clone of https://github.com/debs-obrien/debbie.codes. Then I got a 45 error message similar to the one in the header, on every existing propname.  See a couple of them in the screen-clip below:

![image](https://user-images.githubusercontent.com/115985591/211520571-b391108f-1458-4447-b34d-8f5b709704ba.png)

### Alle the vue-files starts with code similar to this:
![image](https://user-images.githubusercontent.com/115985591/211520953-01fd8fe3-e015-46b8-b959-a56f2594de74.png)

### The repo also has a types.ts-file, who starts with this code:
![image](https://user-images.githubusercontent.com/115985591/211524116-ba02334f-bbe6-47ca-8c4f-07596a6af6fc.png)
See also the files in the repo of [Debbie O'Brien.](https://github.com/debs-obrien/debbie.codes)

**Can someone Typescript-expert help me with this issue? Please!**

**Comment by ndragun92** [comment, marked as answer]
I had some issues with  template validations in my IDE so u can disable it in tsconfig I think

```
{
  "compilerOptions": ...
  "vueCompilerOptions": {
    "experimentalDisableTemplateSupport": true
  }
}
```

**Reply by ghost** [reply]
Thanks a lot for trying to help me!
There is propably something I don't understand with what your are trying to tell me.
When I tried to insert your code into my tsconfig-file,  I got this error in VS-Code:
![image](https://user-images.githubusercontent.com/115985591/213682531-94dea117-1b26-4249-b15a-e2e3d223fa1f.png)

**Reply by dsvgl** [reply]
The options object needs to be one level up, like so:
```json
{
  "compilerOptions": {
  },
  "vueCompilerOptions": {
    "skipTemplateCodegen": true
  }
}
```

Btw: bespoken option was renamed to `skipTemplateCodegen`

**Reply by ghost** [reply]
Thanks a lot! 
This code in tsconfig is working:
```
{
  "compilerOptions": {
  },
  "vueCompilerOptions": {
    "skipTemplateCodegen": true
  }
}
```
45 errors have now disappeared:
![image](https://user-images.githubusercontent.com/115985591/213686878-cb176a7f-71dd-40e7-a654-cf2c307eea59.png)
***Thanks again!***
Finally I can continue to use this wonderful template from Debbie:)

**Reply by ndragun92** [reply]
Great glad to hear that :)

**Comment by akasection** [comment]
I feel like `vue-tsc` somehow didn't pick up your `~/types` path, in which the `vue-tsc` itself loads the wrong `tsconfig`

how about try `<pnpm/yarn/npm run> nuxi typecheck` ? (though it's still using vue-tsc under the hood)
